target/linux/bcm63xx/base-files/etc/board.d/01_leds

```diff
@@ -43,6 +43,9 @@ comtrend,vr-3032u|\
 huawei,hg253s-v2)
 	ucidef_set_led_usbdev "usb" "USB" "$model:green:usb" "1-1"
 	;;
+d-link,dsl-2750u-c1)
+	ucidef_set_led_usbdev "usb" "USB" "dsl-2750u-c1:green:usb" "1-1"
+	;;
```

Comment:
Just add your device to the block directly above, as $model will evaluate to dsl-2750u-c1.